cmd/argocd/commands/login.go

```diff
@@ -331,13 +331,7 @@ func oauth2Login(
 	}
 	fmt.Printf("Performing %s flow login: %s\n", grantType, url)
 	time.Sleep(1 * time.Second)
-	if ssoLaunchBrowser {
-		fmt.Printf("Opening default browser for authentication\n")
-		err = open.Start(url)
-		errors.CheckError(err)
-	} else {
-		fmt.Printf("To authenticate, copy-and-paste the following URL into your preferred browser: %s\n", url)
-	}
+	ssoBrowserFlow(url, ssoLaunchBrowser)
```

Comment:
lets rename the function ssoAuthenticationFlow

Reply:
I've renamed it!